src/github.com/verizon/nelson/util.go

```diff
@@ -42,7 +42,21 @@ func AugmentRequest(c *gorequest.SuperAgent, cfg *Config) *gorequest.SuperAgent
 		Timeout(60*time.Second).
 		Retry(3, 1*time.Second, http.StatusBadGateway, http.StatusInternalServerError).
 		SetCurlCommand(globalEnableCurl).
-		SetDebug(globalEnableDebug)
+		SetDebug(globalEnableDebug).
+		RedirectPolicy(func(req gorequest.Request, via []gorequest.Request) error {
+			// Copy the Cookie on redirect if the hosts match
```

Comment:
My understanding is that [net/http client (line 633)](https://golang.org/src/net/http/client.go) should handle this, but it's not.  

We're being sloppy about case insensitivity of hosts and the HTTP spec, but I think this is enough to make the feature work without compromising the Nelson API or accidentally spewing credentials in case Nelson starts redirecting somewhere else.